components/integrations/bots/bots.test.tsx

```diff
@@ -87,21 +87,17 @@ describe('components/integrations/bots/Bots', () => {
     });
 
     it('bot owner tokens', () => {
-        const bot1 = TestHelper.fakeBot();
+        const bot1 = TestHelper.getBotMock({user_id: '1', owner_id: '1'});
         const bots = {
             [bot1.user_id]: bot1,
         };
 
-        const owner = {
-            user_id: 'owner',
-        };
+        const owner = TestHelper.getUserMock({id: bot1.owner_id});
 
-        const user = {
-            id: bot1.user_id,
-        };
+        const user = TestHelper.getUserMock({id: bot1.user_id});
 
         const passedTokens = {
-            id: 'token',
+            id: TestHelper.getUserAccessTokenMock(),
```

Comment:
This token needs to be a UserAccessToken type object. Created a helper function for mocking this.